# Parameters for the SLAM node.

# Frequency at which we want to operate the SLAM ROS node.
node_loop_rate: 50

# Topic in which the IMU measurements are being received.
imu_meas_topic: /rexrov/imu

# Measurement noise characterisicts:
# TODO(tonioteran) UNINFORMED GUESSES FOR EVERYTHING.
# --Prior factors:
# Standard deviation of the prior information's translation components [m].
prior_pos_stddev: 0.0001
# Standard deviation of the prior information's rotation components [deg].
prior_rot_stddev: 0.000001
# --IMU measurements:
# Standard deviation for the linear acceleration measurements [???].
imu_accel_stddev: 0.001
# Standard deviation for the angular velocity measurements [???].
imu_omega_stddev: 0.001

# SLAM estimate publishers.
dead_reckoning_topic: /slam/dead_reckoning
dead_reckoning_frame_id: map
